gfx/video_crt_switch.c

```diff
@@ -120,11 +120,18 @@ static void set_aspect(videocrt_switch_t *p_switch, unsigned int width,
       patched_width = width;
       patched_height = height;
    }
-   if (srm_isstretched && srm_width > 0)
+   if (srm_width >= 1920)
    {
-      srm_xscale = srm_width/width;
-      srm_yscale = srm_height/height;
+      srm_xscale = (float)srm_width/width;
+      RARCH_LOG("[CRT]: Super resolution detected. Fractal scaling @ X:%f Y:%d \n", srm_xscale, (int)srm_yscale);
+   }
+   else if (srm_isstretched && srm_width > 0 ){
+      srm_xscale = (float)srm_width/width;
+      srm_yscale = (float)srm_height/height;
+      RARCH_LOG("[CRT]: Resolution is stretched. Fractal scaling @ X:%f Y:%f \n", srm_xscale, srm_yscale);
    }
+   else
+      RARCH_LOG("[CRT]: SR integer scaled  X:%d Y:%d \n",srm.x_scale, srm.y_scale);
```

Comment:
You probably want to substitute `srm.x_scale` and `srm.y_scale` with `srm_xscale` and `srm_yscale` respectively and either change format to `%f` or add cast to `(int)`.